src/components/LocaleLink.js

```diff
@@ -20,7 +22,12 @@ const LocaleLink = ({ children, to, locale, ...other }) =>
         locale = defaultLocale
       }
 
-      localeTo = localeTo.replace('/' + defaultLocale + '/', '/')
+      // strip any existing locale from the pathname
+      localeTo = localeTo.replace(/^\/\w{2}\//, "/")
```

Comment:
... I guess the alternative is either doing a GraphQL query + props in a context somewhere, or testing against all ISO locales...

Reply:
Is it bad that it catches non-locale urls? I think they will be ok since `LocaleLink` is only used for internal links. The regex should only catch any two letter phrase between two backlashes at the beginning of the path. So the non-locale urls will just pass through. I don't think we'd create any two letter subdirectory? However, I could be very well missing something.

Reply:
You're right, it probably won't... I'm excited for the day we come up with a two-letter content type (`id`?) and break this, but I guess it's fine for now. Comments might be nice

Reply:
I'll try to update this to only test for our supported locales. Lol I think I was just maybe a teeny bit traumatized by our last attempt at that. 🙈 I'll also add more comments!

Reply:
Updated!